### PR TITLE
[plugin.audio.eco99music] v0.0.1

### PR DESCRIPTION
**Pull request by nire0510**
Thousands of premium playlists based on users mood, activity, location and more
(Official Kodi add-on of Eco 99FM radio station)

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.audio.eco99music/addon.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.beautifulsoup4" version="4.5.3"/>
+        <import addon="script.module.requests" version="2.12.4"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="he">אלפי פלייליסטים וסטים מוזיקליים, ערוכים ומוכנים להאזנה, ומותאמים לכל מצב, בכל מקום ובכל שעה</summary>
```
Comment (enen92) [inline plugin.audio.eco99music/addon.xml]: Please use `he_il`

File: plugin.audio.eco99music/addon.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        <summary lang="en_GB">Thousands of premium playlists based on users mood, activity, location and more</summary>
+        <description lang="he">
```
Comment (enen92) [inline plugin.audio.eco99music/addon.xml]: `he_il`

File: plugin.audio.eco99music/addon.py [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+import urlparse
+import xbmcaddon
+import xbmcgui
+import xbmcplugin
+import requests
+from bs4 import BeautifulSoup
+import re
+
+
+def build_url(query):
```
Comment (enen92) [inline plugin.audio.eco99music/addon.py]: This is a minor comment and you are not required to change it. Normally it is a bad practice to use methods with only one or two lines of code

**Comment by nire0510** [comment]
thanks @enen92 - language code changed to he_il

**Comment by enen92** [comment]
thanks, merged

**Pull request merged**